open_spiel/python/CMakeLists.txt

```diff
@@ -153,6 +153,7 @@ set(PYTHON_TESTS ${PYTHON_TESTS}
   utils/lru_cache_test.py
   utils/spawn_test.py
   pytorch/rcfr_pytorch_test.py
+  pytorch/dqn_test.py
```

Comment:
If you remove dqn_test.py then make sure to change this to pytorch/dqn_pytorch_test.py

Reply:
Yes, I changed!